src/SFA.DAS.Commitments.Database/Tables/JobProgress.sql

```diff
@@ -0,0 +1,9 @@
+﻿-- Constrained to single row, see https://stackoverflow.com/questions/3967372/sql-server-how-to-constrain-a-table-to-contain-a-single-row
+
+CREATE TABLE [dbo].[JobProgress]
```

Comment:
Is JobProgress a bit too generic a name?

Reply:
It was purposely kept generic, so that it could be reused by new web jobs. They could add a column to store their own progress.